scripts/href-remap.js

```diff
@@ -0,0 +1,26 @@
+import { JSDOM } from 'jsdom';
+import isAbsoluteUrl from 'is-absolute-url';
+
+
+export function remap404Hrefs (html, baseUrl) {
+  const jsdom = new JSDOM(html);
+  const win = jsdom.window;
+
+  // Remap href attribute
+  for (let tag of win.document.querySelectorAll('*[href]')) {
+    const currValue = tag.attributes.href.value;
+    if (currValue && !isAbsoluteUrl(currValue) && !currValue.startsWith('//')) {
+      tag.attributes.href.value = `${baseUrl}${currValue}`;
+    }
+  }
+
+  // Remap src attribute
+  for (let tag of win.document.querySelectorAll('*[src]')) {
```

Comment:
we cold do it in a single loop `[...win.document.querySelectorAll('*[href]'), ...win.document.querySelectorAll('*[src]')]` and loop ['src', 'href']